kedro/runner/runner.py

```diff
@@ -406,6 +406,7 @@ def _run_node_sequential(
     return node
 
 
+# pylint: disable=too-many-locals
```

Comment:
Does it make sense to make use of the fact that `save_futures` is a set, containing exactly the same elements as the keys of `future_dataset_mapping` and only use one local variable, thus having not too many local variables in this function?

Reply:
You're right about this, good catch, fixed in [0482020](https://github.com/kedro-org/kedro/pull/2159/commits/0482020abd9e3e01d205ab08cd75ea32bcbff2be).